api/notd/block_processor.py

```diff
@@ -185,23 +200,23 @@ async def _process_erc721_single_event(self, event: LogReceipt) -> List[Retrieve
         retrievedEvents = [RetrievedEvent(transactionHash=transactionHash, registryAddress=registryAddress, fromAddress=fromAddress, toAddress=toAddress, operatorAddress=None, amount=1, tokenId=tokenId, tokenType='erc721')]
         return retrievedEvents
 
-    async def process_transaction(self, transaction: TxData, retrievedEvents: List[RetrievedEvent]) -> List[RetrievedTokenTransfer]:
+    async def process_transaction(self, transaction: TxData, retrievedEvents: List[RetrievedEvent], wethValues: List[tuple]) -> List[RetrievedTokenTransfer]:
```

Comment:
typing isnt good here for wethValues. would suggest a better name to wethTransfers or something similar